config.json

```diff
@@ -107,9 +107,15 @@
       "topics": []
     },
     {
-      "difficulty": 1,
+      "difficulty": 2,
       "slug": "grains",
-      "topics": []
+      "topics": [
+        "Control-flow (if-else statements)",
+        "Bitwise operations",
+        "Mathematics",
+        "Floating-point numbers",
```

Comment:
uncertain of this one, given that all answers fit in unsigned integers. But did you see someone use floating-point numbers? If so, OK.